src/components/views/elements/ServerPicker.tsx

```diff
@@ -81,7 +82,9 @@ const ServerPicker = ({ title, dialogTitle, serverConfig, onServerConfigChange }
         </span>;
     }
 
-    return <div className="mx_ServerPicker">
+
+    return config.showHomeServerDetail &&
```

Comment:
This need to change to handle that showHomeServerDetail should default to true if showHomeServerDetail is not in the config